### PR TITLE
[Doc Improvement][Channel-and-group-conversations]

### PR DESCRIPTION
**Pull request by v-shalinir**
Merge @ user mention in Build cards > Overview page.
Removed note for higher cloud limitation? It is enabled for all tenants.

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/how-to/conversations/channel-and-group-conversations.md](https://github.com/MicrosoftDocs/msteams-docs/blob/03dd4d1968d09eb53dd8174c2302c01e91c02dcf/msteams-platform%2Fbots%2Fhow-to%2Fconversations%2Fchannel-and-group-conversations.md) | 90 | 12 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/3fe1c2ed-7411-4a4a-90d9-fff38fabfa54) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/how-to/conversations/channel-and-group-conversations.md](https://github.com/MicrosoftDocs/msteams-docs/blob/4f96a07404c2f62f11cfffe6ea2684a5c66a9b8e/msteams-platform%2Fbots%2Fhow-to%2Fconversations%2Fchannel-and-group-conversations.md) | 90 | 11 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/0a0d174e-898d-4887-a1c5-a127dadb72ce) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/includes/bots/user-mention.md](https://github.com/MicrosoftDocs/msteams-docs/blob/4f96a07404c2f62f11cfffe6ea2684a5c66a9b8e/msteams-platform%2Fincludes%2Fbots%2Fuser-mention.md) | 93 | 1 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/71c29932-fe86-4726-808e-84635dbd1b63) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/task-modules-and-cards/what-are-cards.md](https://github.com/MicrosoftDocs/msteams-docs/blob/4f96a07404c2f62f11cfffe6ea2684a5c66a9b8e/msteams-platform%2Ftask-modules-and-cards%2Fwhat-are-cards.md) | 96 | 11 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/de7c6501-2120-4b20-9923-14a00c14faaf) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/how-to/conversations/channel-and-group-conversations.md](https://github.com/MicrosoftDocs/msteams-docs/blob/53b7c971fbd98060327dd917fd99f84c0a3ddacf/msteams-platform%2Fbots%2Fhow-to%2Fconversations%2Fchannel-and-group-conversations.md) | 90 | 11 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/3f86243f-3a3b-47dd-8788-4b641016a8c8) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/includes/bots/user-mention.md](https://github.com/MicrosoftDocs/msteams-docs/blob/53b7c971fbd98060327dd917fd99f84c0a3ddacf/msteams-platform%2Fincludes%2Fbots%2Fuser-mention.md) | 93 | 1 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/35d7f5af-c73f-41f4-946c-9fb7b9128820) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/task-modules-and-cards/what-are-cards.md](https://github.com/MicrosoftDocs/msteams-docs/blob/53b7c971fbd98060327dd917fd99f84c0a3ddacf/msteams-platform%2Ftask-modules-and-cards%2Fwhat-are-cards.md) | 96 | 11 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/f729ec7e-c510-4033-a473-e7d3aea8dfac) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/how-to/conversations/channel-and-group-conversations.md](https://github.com/MicrosoftDocs/msteams-docs/blob/1581609665f69c02a2d899f280993c2c634254b8/msteams-platform%2Fbots%2Fhow-to%2Fconversations%2Fchannel-and-group-conversations.md) | 90 | 11 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/216e7860-a814-4030-8235-a82649603c80) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/includes/bots/user-mention.md](https://github.com/MicrosoftDocs/msteams-docs/blob/1581609665f69c02a2d899f280993c2c634254b8/msteams-platform%2Fincludes%2Fbots%2Fuser-mention.md) | 93 | 1 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/0d468e28-8248-478f-b450-94ab0c4050c2) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/task-modules-and-cards/what-are-cards.md](https://github.com/MicrosoftDocs/msteams-docs/blob/1581609665f69c02a2d899f280993c2c634254b8/msteams-platform%2Ftask-modules-and-cards%2Fwhat-are-cards.md) | 96 | 11 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/3b82486e-c7c0-4483-b449-677ea8bb493d) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/how-to/conversations/channel-and-group-conversations.md](https://github.com/MicrosoftDocs/msteams-docs/blob/967782989b97373ec3e7668ec032a98194f99c50/msteams-platform%2Fbots%2Fhow-to%2Fconversations%2Fchannel-and-group-conversations.md) | 87 | 17 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/fe936905-b217-4fc2-af87-5a9a31183355) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/includes/bots/user-mention.md](https://github.com/MicrosoftDocs/msteams-docs/blob/967782989b97373ec3e7668ec032a98194f99c50/msteams-platform%2Fincludes%2Fbots%2Fuser-mention.md) | 100 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/6fc3f504-2130-48cf-a65c-b6cb6cdb71c2) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/how-to/conversations/channel-and-group-conversations.md](https://github.com/MicrosoftDocs/msteams-docs/blob/5c34c3560817bf671135e6c6b827fe0b71db544d/msteams-platform%2Fbots%2Fhow-to%2Fconversations%2Fchannel-and-group-conversations.md) | 87 | 17 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/f500a1cb-3d1a-45bd-beaa-9c469277b47e) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/includes/bots/user-mention.md](https://github.com/MicrosoftDocs/msteams-docs/blob/5c34c3560817bf671135e6c6b827fe0b71db544d/msteams-platform%2Fincludes%2Fbots%2Fuser-mention.md) | 100 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/3ae82c70-4f81-444e-9977-58778d01efce) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/bots/how-to/conversations/channel-and-group-conversations.md [hunk 1]
```diff
@@ -384,6 +386,41 @@ The following table lists the throttling limits for tag mentions in a bot:
 * Tag mentions aren't supported in connectors.
 * Tag mentions don't support the invoke flow in a bot.
 
+### Support for Microsoft Entra Object ID and UPN in user mention
```
Comment (limamicro) [inline msteams-platform/bots/how-to/conversations/channel-and-group-conversations.md:389]: As suggested in the group chat, how about we move this section under User mention?
#### User mention
##### Support for Microsoft Entra Object ID and UPN

File: msteams-platform/includes/bots/user-mention.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+---
+title: User Mention
+description: With this learning module, learn how your bot can mention other users in messages posted in channels.
+localization_priority: Normal
+ms.topic: overview
+ms.author: surbhigupta12
+ms.date: 11/14/2024
+---
+
+Bots with Adaptive Cards support user mention IDs, such as Microsoft Entra Object ID and User Principle Name (UPN) in addition to the existing IDs. Incoming Webhooks start to support user mention in Adaptive Card with the Microsoft Entra Object ID and UPN.
```
Comment (limamicro) [inline msteams-platform/includes/bots/user-mention.md:10]: Please remove "with Adaptive Cards". It should be: "Bots support user mention IDs, such as Microsoft.....".

**Review by limamicro — CHANGES_REQUESTED** [review]

File: msteams-platform/bots/how-to/conversations/channel-and-group-conversations.md (hunk 1 above)
Comment (v-shalinir) [reply msteams-platform/bots/how-to/conversations/channel-and-group-conversations.md:389]: @limamicro As suggested, I've moved the section to flow as:
- User mention
- Support for Microsoft Entra Object ID and UPN
- Tag mention

Please review and let me know if flow is okay now.

File: msteams-platform/includes/bots/user-mention.md (hunk 2 above)
Comment (v-shalinir) [reply msteams-platform/includes/bots/user-mention.md:10]: @limamicro I've updated the input.

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/how-to/conversations/channel-and-group-conversations.md](https://github.com/MicrosoftDocs/msteams-docs/blob/75fbae293cbd660ca08e8337811a4b8c1def2b10/msteams-platform%2Fbots%2Fhow-to%2Fconversations%2Fchannel-and-group-conversations.md) | 88 | 17 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/6f9506ec-1a5d-489c-83e4-ca0b56883b32) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/includes/bots/user-mention.md](https://github.com/MicrosoftDocs/msteams-docs/blob/75fbae293cbd660ca08e8337811a4b8c1def2b10/msteams-platform%2Fincludes%2Fbots%2Fuser-mention.md) | 100 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/376b31b2-0e30-4c52-893d-3f23fb7b73e4) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/task-modules-and-cards/what-are-cards.md](https://github.com/MicrosoftDocs/msteams-docs/blob/75fbae293cbd660ca08e8337811a4b8c1def2b10/msteams-platform%2Ftask-modules-and-cards%2Fwhat-are-cards.md) | 96 | 11 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/12aacbcd-40e6-49d2-8f24-cc985959c9a0) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/how-to/conversations/channel-and-group-conversations.md](https://github.com/MicrosoftDocs/msteams-docs/blob/ce92dfba5e4241495c1606cbd2e5046ad6733f77/msteams-platform%2Fbots%2Fhow-to%2Fconversations%2Fchannel-and-group-conversations.md) | 88 | 17 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/f4a96dd7-b21f-4a44-bff7-8039292e73a6) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/includes/bots/user-mention.md](https://github.com/MicrosoftDocs/msteams-docs/blob/ce92dfba5e4241495c1606cbd2e5046ad6733f77/msteams-platform%2Fincludes%2Fbots%2Fuser-mention.md) | 100 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/8f8f8afe-4166-4e30-8f1c-8c1bd8f57265) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/task-modules-and-cards/what-are-cards.md](https://github.com/MicrosoftDocs/msteams-docs/blob/ce92dfba5e4241495c1606cbd2e5046ad6733f77/msteams-platform%2Ftask-modules-and-cards%2Fwhat-are-cards.md) | 96 | 11 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/9ca5b97e-d516-489f-aee0-09301e05793c) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by surbhigupta12 — APPROVED** [review]

**Pull request merged**